README.md

```diff
@@ -59,10 +59,11 @@ Currently, only the Windows version of the game is supported (due to the Cutscen
   - Open the game launcher and set language, change resolution and set to windowed or borderless mode.
   - Launch the base game (`FFX.exe`). Press `Escape` and go into `System Settings > Key Bindings > Controller` and set the `Talk/Examine/Confirm` option to `A`.
 * Make sure that there are no physical XBox/Playstation controllers connected. This is because the virtual gamepad needs to be controller 1 for the TAS to work.
-* Make sure the correct settings for the run are set in `vars.py`:
-  - Change `self.setSeed = True` to `self.setSeed = False`
-* Make sure the correct GameState for your run is set in `main.py`. For a new game, use `GameState = "none"` and `step_counter = 1`. Comment out any other assignment at the top of `main.py`.
-* In order for other GameStates to work, put the contents of `TAS Saves` into `%userprofile%/Documents/SQUARE ENIX/FINAL FANTASY X&X-2 HD Remaster/FINAL FANTASY X/`.
+* Make sure the correct settings for the run are set in `config.yaml`:
```

Comment:
Once we update the yaml file to require all of these keys (while also excluding the yaml file from source control), we can drop this (as these will be the defaults). For now though, it's fine to keep these in.